docs/community.rst

```diff
@@ -1,18 +1,22 @@
 Community Resources
 ===================
 
-Below are a few community resources about Rally. If you find an interesting article, talk or custom tracks, `raise an issue <https://github.com/elastic/rally/issues>`_  or `open a pull request <https://github.com/elastic/rally/pulls>`_.
+Below are a few community resources about Rally. If you find an interesting article, talk, or custom tracks, `raise an issue <https://github.com/elastic/rally/issues>`_  or `open a pull request <https://github.com/elastic/rally/pulls>`_.
 
-Talks
------
+Videos
+------
 
 .. raw:: html
 
    <iframe width="560" height="315" src="https://www.youtube.com/embed/HriBY2zoChw" frameborder="0" allowfullscreen></iframe>
 
 .. raw:: html
 
-   <iframe width="560" height="315" src="https://www.youtube.com/embed/Fm7fxE4kxPI" frameborder="0" allowfullscreen></iframe>
```

Comment:
whatever this video was, I couldn't see it
<img width="652" alt="image" src="https://user-images.githubusercontent.com/7477841/113720581-cd75ce80-96bc-11eb-9546-e9e444b73c77.png">